src/EnergyPlus/ZoneAirLoopEquipmentManager.cc

```diff
@@ -690,14 +690,8 @@ namespace ZoneAirLoopEquipmentManager {
                 if (InNodeNum > 0) {
                     MassFlowRateMaxAvail = state.dataLoopNodes->Node(InNodeNum).MassFlowRateMaxAvail;
                     MassFlowRateMinAvail = state.dataLoopNodes->Node(InNodeNum).MassFlowRateMinAvail;
-                    AirLoopNum = state.dataDefineEquipment->AirDistUnit(AirDistUnitNum).AirLoopNum;
-                    if (AirLoopNum > 0) {
-                        DesFlowRatio = state.dataAirLoop->AirLoopFlow(AirLoopNum).SysToZoneDesFlowRatio;
-                    } else {
-                        DesFlowRatio = 1.0;
-                    }
```

Comment:
Not needed any longer.

Reply:
Depending on the value of the `IsConstLeakageRate` flag either a constant leakage flow rate or a proportional leakage flow rate will be calculated. Four `VAV` air terminals will use a constant leakage flow rate (assuming there is duct static pressure reset), and two `CV` air terminals will use a proportional leakage flow rate. Currently, only `six` air terminals support the air leakage model.